Basic/Pod/FAQ.pod

```diff
@@ -804,7 +804,7 @@ very useful !
 
 Most people will try to form new piddles from old piddles
 using some variation over the theme: 
-C<$a =  pdl([$b, 0, 2])> , but this does not work. The way to
+C<$x =  pdl([$y, 0, 2])> , but this does not work. The way to
```

Comment:
Again not a problem with your changes, but this example actually does work when it is advertised not to.  Probably it doesn't work in the way a novice user would expect, though.

Reply:
If you can spell out how to fix the example I'll be pleased to make the change. Your PDL knowledge far outweighs mine :-)